<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketLayersWidget</class>
 <widget class="QWidget" name="RocketLayersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>107</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#controlsFrame {
	background-color: white;
	border-top: 1px solid rgb(207, 207, 207);
}

QLabel, QPushButton {
	color: rgb(80, 80, 80);
	font: 12pt &quot;Arial&quot;;
	font-weight: bold;
}

QLabel#labelLayersTitle {
	font-family: &quot;Arial&quot;;
	font-size: 22px;
	color: rgb(65, 65, 65);
}

QPushButton {
	font-family: &quot;Arial&quot;;
	font-size: 16px;
	min-height: 25px;
	border: 1px solid rgba(104, 104, 104, 150);
	border-radius: 3px;
	color: rgb(66, 66, 66);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(233, 233, 233, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:disabled {
	color: rgb(172, 172, 172);
}

QPushButton:checked {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 233, 0, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton::hover {
	border: 1px solid rgba(104, 104, 104, 225);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.7 rgba(238, 238, 238, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton::pressed {
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(238, 238, 238, 255), stop:0.5 rgba(255, 255, 255, 255));
}

QPushButton#buttonClose, #downloadAllButton, #hideAllButton {
	background-color: transparent;
	color: rgb(80, 80, 80);
	border: 1px solid grey;
	border: 0px;
	border-radius: 3px;
	font-family: &quot;Arial&quot;;
	font-size: 13px;
	font-weight: bold;
	min-height: 25px;
	max-height: 25px;
	min-width: 75px;
	max-width: 75px;
	padding-left: 5px;
	padding-right: 5px;
}

QPushButton#downloadAllButton, #hideAllButton {
	max-width: 125px;
	color: rgb(70, 70, 70);
}

QPushButton#buttonClose::hover, #downloadAllButton::hover, #hideAllButton::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(232, 234, 233, 255), stop:1 rgba(225, 226, 227, 255));
	color: rgb(60, 60, 60);
	border: 1px solid grey;
}

QPushButton#buttonClose::pressed, #downloadAllButton::pressed, #hideAllButton::pressed {
	background-color: rgba(200, 200, 200);
	border: 1px solid grey;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>477</width>
           <height>69</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="layersLayout">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="controlsFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="labelLayersIcon">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="RocketPlugin.qrc">:/images/icon-layers-big.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="downloadAllButton">
        <property name="text">
         <string>Download All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hideAllButton">
        <property name="text">
         <string>Hide All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClose">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="RocketPlugin.qrc">
          <normaloff>:/images/icon-check.png</normaloff>:/images/icon-check.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RocketPlugin.qrc"/>
 </resources>
 <connections/>
</ui>
